src/common/snippets/include/snippets/pass/tokenization.hpp

```diff
@@ -48,8 +57,23 @@ class EnumerateNodes : public ov::pass::ModelPass {
  */
 class SnippetsTokenization : public ov::pass::ModelPass {
 public:
+    /**
+     * @interface Config
+     * @brief Allow to adjust tokenization passes using the corresponding Configs
+     * @ingroup snippets
+     */
+    struct Config {
```

Comment:
I think it's scalable approach. If you want to adjust other tokenization passes, you can add the own `Config`. I don't want to have all setting parameters in common `Config` because this config will have all parameters and it's not clear which from them are for `MHATokenization`, which are from other passes.

Reply:
UPD: Discussed offline. Refactored to have only one Config with the common conditions.